package.json

```diff
@@ -7,6 +7,8 @@
     "clean": "rm -rf dist",
     "prebuild": "graphql-codegen",
     "build": "ng build --prod --localize",
+    "build-dev": "ng build --localize",
+    "build-stage": "ng build --configuration=staging --localize",
```

Comment:
Please take care to apply the `prebuild` script also for the new `build-*` jobs.

Reply:
done